js/script.js

```diff
@@ -1,15 +1,162 @@
 // Javascript Code.
-$(document).ready(function () {
+//Variables where I keep every element of json
+ptitle = [];
+pcontent = [];
+pdate = [];
+$(document).ready(function() {
+  
```

Comment:
Truly good work, the code structure is pretty clean and easy to read, good use of comments when needed and indentation really helps out, :)